tests/conformance/test_conformance.py

```diff
@@ -485,9 +672,18 @@ def blob_create_resumable_upload_session(client, _preconditions, **resources):
         blobwriter_write,
         blob_create_resumable_upload_session,
     ],
-    "storage.objects.patch": [blob_patch],
+    "storage.objects.patch": [
+        blob_patch,
+        object_acl_save,
```

Comment:
Same as for bucket ACLs above-- this should be an entirely separate route than storage.objects.patch I believe.